config/default/SessionStateService.conf.php

```diff
@@ -16,4 +17,5 @@
  *
  * Copyright (c) 2015 (original work) Open Assessment Technologies SA;
  */
-return new oat\taoQtiTest\models\SessionStateService(array());
+
+return new oat\taoQtiTest\models\SessionStateService([]);
```

Comment:
add this to use section:

`use oat\taoQtiTest\models\SessionStateService;`

```suggestion
return new oat\taoQtiTest\models\SessionStateService([]);
```